Sources/CustomDump/Conformances/Swift.swift

```diff
@@ -1,9 +1,27 @@
+import Foundation
+
 extension Character: CustomDumpRepresentable {
   public var customDumpValue: Any {
     String(self)
   }
 }
 
+#if (swift(>=5.7) && (os(iOS) || os(tvOS) || os(watchOS))) || (os(macOS) && swift(>=5.7.1))
```

Comment:
Starting with an easy conformance for Apple platforms. Could take a custom conformance for Linux/Wasm/Windows or wait for the new open source Foundation to close the gap.